docs/design/video-template.ep

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
```

Comment:
Add please actions, notations and comments where needed.
Here is a [pdf file](https://www.dropbox.com/s/gsxjje1fns4goz8/InteractiveSketchingNotation_0.1.pdf?dl=0), which will help you with it.